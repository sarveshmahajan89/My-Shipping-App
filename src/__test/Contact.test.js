import React from 'react';

import Contact from '../component/Contact.js'

it('Contact renders correctly', () => {

    expect(<Contact />).toMatchSnapshot();
});
