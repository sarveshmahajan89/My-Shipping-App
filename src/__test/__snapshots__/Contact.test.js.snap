// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `<App />`;

exports[`Contact renders correctly 1`] = `<Contact />`;
